src/one_dragon/base/notify/push.py

```
@@ -750,10 +846,10 @@ def smtp(self, title: str, content: str, image: Optional[BytesIO]) -> None:
         try:
             smtp_server = (
                 smtplib.SMTP_SSL(self.get_config("SMTP_SERVER"))
-                if self.get_config("SMTP_SSL") == "true"
+                if src(self.get_config("SMTP_SSL")).lower() == "true"
```

Ruff (0.12.2)

Undefined name `src`

(F821)